docs/docs/Configuration.md

```diff
@@ -8,7 +8,58 @@ index: 5
 Fantomas ships with a series of format options.
 These can be stored in an [.editorconfig](https://editorconfig.org/) file and will be picked up automatically by the commandline tool.
 
-A default .editorconfig file would look like
+<!-- 
+    This index can be auto-generated with VS Code's Markdown All in One extension.
+    The ToC will be updated-on-save, or can be generated on command by using
+    Ctrl-Shift-P: "Create table of contents". 
+    More info: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one#table-of-contents 
+-->
+## Index<!-- omit in toc -->
+- [Configuration](#configuration)
```

Comment:
This anchor is wrong, should be `#Configuration`.

Reply:
Actually, it's `#configuration`, see https://github.com/fsprojects/fantomas/blob/b11b645a997a0baa2dea51a7c1aba570ce9dceca/docs/docs/Configuration.md#configuration

Reply:
I’d want to, I just didn’t know how (at least not trivially). Looks like fsdocs does not read the added anchors. They were there to prevent ambiguity in the auto creation of header anchors, which apparently each markdown reader does differently.

So, the code explicitly has the lowercase `c` here, fsdocs just totally ignores it…